README.md

```diff
@@ -330,6 +330,32 @@ prompt = "Face of a yellow cat, high resolution, sitting on a park bench"
 image = pipe(prompt=prompt, image=init_image, mask_image=mask_image).images[0]
 ```
 
+### Weight Reuse
+
+It's possible to reuse weights between pipelines. For example, if you wanted to use a StableDiffusionPipeline and a StableDiffusionImg2ImgPipeline, you could initialize the weights for the first and apply them to the latter:
+```python
+# make sure you're logged in with `huggingface-cli login`
+from diffusers import StableDiffusionPipeline, StableDiffusionImg2ImgPipeline
+
+sd = StableDiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5")
+
+# Note that each subcomponent can be added or removed separately.
+sd_img2img = StableDiffusionImg2ImgPipeline.from_pretrained(
```

Comment:
```suggestion
sd_img2img = StableDiffusionImg2ImgPipeline.from_pretrained(**sd_img2img.components)
```
Should do the same in just one line -> should we  maybe advertise this instead? :-) 
See https://huggingface.co/docs/diffusers/api/diffusion_pipeline#diffusers.DiffusionPipeline.components